Traducir.Web/src/rules/nonUndefinedReactNodeOnRenderRule.ts

```diff
@@ -0,0 +1,39 @@
+import * as Lint from "tslint";
+import * as ts from "typescript";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FailureString: string = `In Component and PureComponent, render() return type should be "NonUndefinedReactNode".`;
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        return this.applyWithWalker(new NonUndefinedReactNodeOnRenderWalker(sourceFile, this.getOptions()));
+    }
+}
+
+// tslint:disable-next-line:max-classes-per-file
```

Comment:
is this comment relevant to the class?

Reply:
Yes, otherwise tslint itself warns about being two classes in the same file.